codescantask/app.input.ts

```diff
@@ -37,4 +37,9 @@ export const SBOM_ENABLED = tl.getInput('sbomEnabled') === 'true';
 export const SBOM_FILEPATH = tl.getInput('sbomFilepath') || "sbom.json" ;
 export const SBOM_TYPE = tl.getInput('sbomType');
 export const POLICIES_HALT_ON_FAILURE = tl.getInput('policiesHaltOnFailure') === 'true';
-export const RUNTIME_CONTAINER = tl.getInput('runtimeContainer') || "ghcr.io/scanoss/scanoss-py:v1.15.0";
+export const RUNTIME_CONTAINER = tl.getInput('runtimeContainer') || "ghcr.io/scanoss/scanoss-py:v1.18.0";
+export const SKIP_SNIPPETS = tl.getInput('skipSnippets') === 'true';
+export const SCAN_FILES = tl.getInput('scanFiles') === 'true';
+export const SCANOSS_SETTINGS = tl.getInput('scanossSettings') === 'true';
```

Comment:
Here maybe we should do a check for the RUNTIME_CONTAINER

We should only allow to pass `SCANOSS_SETTINGS` and `SETTINGS_FILE_PATH` if the scanoss.py version supports it

Reply:
Every Azure task has pingued an scanoss-py version. We shouldn´t have to check for the RUNTIME_CONTAINER